diesel_migrations/migrations_macros/src/embed_migrations.rs

```diff
@@ -46,6 +46,10 @@ pub fn derive_embed_migrations(input: &syn::DeriveInput) -> quote::Tokens {
             fn revert(&self, _conn: &SimpleConnection) -> Result<(), RunMigrationsError> {
                 unreachable!()
             }
+
+            fn file_path(&self) -> Option<&Path> {
+                Some(Path::new(self.version))
```

Comment:
I think instead of returning a fake path here, we should fix the formatting to just print the version if the path is not set.